### PR TITLE
Add tryout section for Choreo Insights API

### PR DESCRIPTION
**Pull request by RAVEENSR**
## Purpose
Add tryout section for Choreo Insights API

## Goals
Add tryout section for Choreo Insights API

## Samples
Screenshot 2021-09-17 at 14 58 31 (2)

### CODE REVIEW

File: en/docs/insights/choreo-insights-api.md [hunk 1]
```diff
@@ -2,11 +2,16 @@
...
 Choreo Insights API is a GraphQL API that allows you to fetch data in a flexible manner by defining the criteria with precision. This guide explains the different ways in which you can fetch data via this API.
 
-Try out the Choreo Insights API from [here.](../graphiql-explorer/)
-
  - **Allowed operations:** [Queries](#query)
  - **Schema defined types:** [Objects](#objects), [Inputs](#inputs), [Enums](#enums) and [Scalars](#scalars)
 
+##Try out
+
+You can try out the Choreo Insights API with your data using GraphQL Explorer from [here.](../graphiql-explorer/)
```
Comment (RukshiW) [inline en/docs/insights/choreo-insights-api.md:10]:
```suggestion
You can try out the Choreo Insights API with your data via the [GraphQL Explorer](../graphiql-explorer/).
```

File: en/docs/insights/choreo-insights-api.md [hunk 2]
```diff
@@ -2,11 +2,16 @@
...
+
+!!! warning
+    GraphQL Explorer is not working in the Safari web browser due to an [issue](https://developer.apple.com/forums/thread/658688) in the browser.
```
Comment (RukshiW) [inline en/docs/insights/choreo-insights-api.md:13]:
```suggestion
You cannot work with the GraphQL Explorer with the Safari web browser at present due to [this issue](https://developer.apple.com/forums/thread/658688) in the browser.
```

**Review by RukshiW — APPROVED** [review]

File: en/docs/insights/choreo-insights-api.md (hunk 1 above)
Comment (RAVEENSR) [reply en/docs/insights/choreo-insights-api.md]: Fixed with 2a243d1

File: en/docs/insights/choreo-insights-api.md (hunk 2 above)
Comment (RAVEENSR) [reply en/docs/insights/choreo-insights-api.md]: Fixed with 2a243d1

**Review by RukshiW — APPROVED** [review]

**Pull request merged**